File: src/assets/usage-examples-code/timers/stop_timer__named/stop_timer_named-1-usage-example.cpp
#include <iostream>
#include "splashkit.h"

int main()
{
    timer myTimer = create_timer("MyTimer");
    start_timer(myTimer);
    delay(5000);
    double elapsedSeconds = timer_ticks(myTimer) / 1000.0;

    // Stop the timer
    stop_timer(myTimer);

    int elapsedSecondsInt = static_cast<int>(elapsedSeconds);
    std::cout << "The timer stopped at " << elapsedSecondsInt << " seconds" << std::endl;

    return 0;
}

File: src/assets/usage-examples-code/timers/stop_timer__named/stop_timer_named-1-usage-example.cs
using System;
using SplashKitSDK;

class Program
{
    static void Main()
    {
        SplashKitSDK.Timer myTimer = SplashKit.CreateTimer("MyTimer");
        SplashKit.StartTimer(myTimer);
        SplashKit.Delay(5000);
        double elapsedSeconds = SplashKit.TimerTicks(myTimer) / 1000.0;
        int elapsedSecondsInt = (int)elapsedSeconds;

        // Stop the timer 
        SplashKit.StopTimer(myTimer);

        Console.WriteLine($"Elapsed time: {elapsedSecondsInt} seconds");
    }
}
